### PR TITLE
Download page: Improve paragraph on image verification

### PR DESCRIPTION
**Pull request by msiism**
Better language, better logic, better link text. ;)

### CODE REVIEW

File: download/index.markdown [hunk 1]
```diff
@@ -16,7 +16,7 @@ An internet connection via Ethernet or WiFi is required for network installation
...
 ### Verifying file integrity and its digital signature
 
-Prior to using any image you're strongly encouraged to validate the
-signatures on the image to ensure they haven't been tampered with.
-The process for validating these keys can be found [in the
-documentation](https://docs.voidlinux.org/installation/index.html#downloading-installation-media).
+It is strongly recommended to validate the integrity and authenticity of any
+downloaded image before using it, to ensure it hasn’t been tampered with.
+[Instructions on how to do that](https://docs.voidlinux.org/installation/index.html#downloading-installation-media)
+are provided in the Void Handbook.
```
Comment (ericonr) [inline download/index.markdown:22]: Maybe `Void Handbook` could be a link to docs.voidlinux.org ?

Comment (ericonr) [inline download/index.markdown:22]: The indentation seems to be a bit all over the place, could you try to fix that?

Comment (msiism) [reply download/index.markdown]: Okay, done.

I've also fixed the URL for the verification instructions.

Comment (msiism) [reply download/index.markdown]: I'm not sure what you mean by that. Could you elaborate?

Comment (ericonr) [reply download/index.markdown]: I had looked at it on my phone. On desktop it's fine.

**Comment by ericonr** [comment]
Please squash into a single commit.

**Pull request merged**